src/lib.rs

```diff
@@ -13,9 +13,16 @@ pub struct Pass {
 }
 
 #[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct Stat {
+    pub name: String,
+    pub cnt: f64,
+}
+
+#[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct Run {
     pub name: String,
     pub passes: Vec<Pass>,
+    pub stats: Vec<Stat>,
```

Comment:
This probably wants to be `#[serde(default)]` or something like that since we might be loading a run without stats (historical data) -- I think we'll panic right now on such a run?